### DISCUSSION TITLE
Seperate ports for website access and connection?

### DISCUSSION THREAD

**Discussion by Juraszka** [immense/Remotely#941] [Q&A]
Hi
I looking how to make seperate ports for website access and port to connection agent-server.
I need to have only specific computer with access to website remotely_address:5000 but with full possibility to connect to computers with agents configured to acceppt connection from server configured e.g. remotely_address:5100

It it possible or is possible to achieve using reverse proxy?

**Comment by rockenrooster** [comment]
I am also wanting to implement this on my own setup. I would like to only expose the Remotely UI to my LAN instead of the Internet like the Client agents.

**Comment by sshnevis** [comment]
i think there is no way to do it. remotely works with the one specific port.
